CHANGELOG.md

```diff
@@ -6,6 +6,40 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.2.0] 2019-10-21
+
+### Added
+
+- Added a new [public plugin interface](pkg/secretless/plugin) for building connector plugins
+- Added a new [public log interface](pkg/secretless/log) for standardizing logging
```

Comment:
nit and not a blocker - might be more appropriate to only link `<name> interface` words rather than the whole `public <name> interface`